rules/sch/parts/PEPPOL-M-T111.sch

```diff
@@ -19,5 +19,11 @@
 				flag="fatal">Specification identifier SHALL start with the value 'urn:fdc:peppol.eu:poacc:trns:invoice_response:3'.</assert>
 	</rule>
 
+	<rule context="cac:Status[cbc:StatusReasonCode='PPD']">
```

Comment:
There is no rule that checks the affinity of codes (PPD only applicable when PD is Response Code). If response codes other than PD are allowed this should be clearly stated in the specification document, otherwise, the schematron rules need to be further updated.

Reply:
QA shows that following additional rules are needed.

1. Check that PPD can only be used along with PD.
2. Check that document reference is included at least once using document type code 38

Propose to add these in a separate branch and merge this branch if other parts are OK.